main.go

```diff
@@ -48,10 +48,55 @@ type slackMessage struct {
 	slackTag
 }
 
+func (s *slackOutput) globalRoutes(fields map[string]interface{}) []decode.NotificationRoute {
+	// All Production OOM killed messages go to slack
+	content, ok := fields["rawlog"]
+	if !ok || content == nil || content == "" {
```

Comment:
yeah, these can probably be simplified into just `!ok || <var> == ""`. The nil check is likely redundant.